modules/layout/react/lib/Layout.tsx

```diff
@@ -53,9 +53,14 @@ const LayoutContainer = styled('div')<LayoutProps>(
   }
 );
 
+export function stripUnit(value: string): number {
+  return parseInt(`${value}`.replace('px', ''), 10);
+}
+
 export default class Layout extends React.Component<LayoutProps> {
   static defaultProps = {
     gutter: canvas.spacing.xs,
+    spacing: stripUnit(canvas.spacing.xs),
```

Comment:
Not 100% sure if this is the correct way to handle the default spacing for a child. What I am attempting to do is have the child spacing have a default value of `canvas.spacing.xs` But with the ability to override it at the `Layout` and `Column` level like this:

```tsx
       <Layout spacing={60}>
          <Layout.Column spacing={0}>
            <Card heading="Custom Spacing (0px)" />
          </Layout.Column>
          <Layout.Column>
            <Card heading="Custom Spacing (60px)" />
          </Layout.Column>
          <Layout.Column>
            <Card heading="Custom Spacing (60px)" />
          </Layout.Column>
        </Layout>
```

I found that if I set the default spacing in `Column` instead of the `Layout` [this check](https://github.com/Workday/canvas-kit/blob/eed62cf9591dc97fd98c305ac96caeb42acc0da9/modules/layout/react/lib/Layout.tsx#L76) would always be true.